packages/v4/src/content/design-guidelines/usage-and-behavior/lists-and-tables/lists-and-tables.md

```diff
@@ -8,20 +8,33 @@ showTitle: true
 PatternFly offers two components for displaying large data sets: [data list](/documentation/react/components/datalist/) and [table](/documentation/react/components/table/). While they satisfy similar use cases, choosing the correct component to use in your design will be dependent on the type of data you need to display.
 
 Use [data lists](#data-lists) when:
-* A flexible layout is more important than arranging information in a grid
-* You want to include active content like a chart
-* Content displayed may vary between rows
+* A flexible layout is more important than arranging information in a grid.
+* You want to include active content like a chart.
+* Content displayed may vary between rows.
 
 Use [tables](#tables) when:
-* Users will want to consume data as a grid (i.e. structured rows and columns)
-* You want column headers
+* Users will want to consume data as a grid (i.e. structured rows and columns).
+* You want column headers.
 
 ## Data lists
 PatternFly supports several variations of the data list component.
 
-### Standard data list
+### How to use
+Think of each row in a data list as a container for some formatted content. In PatternFly 4, data list rows can accept any [layout](/documentation/react/layouts/bullseye) supported by the design system as long as all rows apply the same layout. Here are some common layouts that may be useful in a data list:
+
+* **[Grid](/documentation/react/layouts/grid):** When you want to display content in a responsive grid.
+* **[Level](/documentation/react/layouts/level):** When you want to justify content evenly over the width of the row.
```

Comment:
> justify content evenly

Is that a common way of referring to what the level layout does? I would think it would say something like "vertically align content".

Reply:
I probably wrote this originally. That's what I thought the Level layout did @mcoker . But if I'm wrong, we can change this. What would you say to describe the Level?

Reply:
Oh nevermind! I forgot it applies "space between" justification of items, while also centering things vertically. If "justify content evenly" conveys that, that's fine. It might also be worth noting that it also vertically aligns items in the center of the row.

FWIW, it's the `space-between` value listed here, if that provides more context - https://developer.mozilla.org/en-US/docs/Web/CSS/justify-content#Values. It applies evenly distributed spacing between items in a row.

> The items are evenly distributed within the alignment container along the main axis. The spacing between each pair of adjacent items is the same. The first item is flush with the main-start edge, and the last item is flush with the main-end edge.